DataLink.tex

```diff
@@ -808,62 +813,56 @@ \section{Service Descriptors}
 metadata resource in the response document to describe each type of
 service that can be used.
 
-The same mechanism can also be used in any VOTable document, such
+Here we describe how to construct a resource that describes a service
+and add it to a VOTable document. This ``service descriptor'' mechanism can also 
+be used in any VOTable document, such
 as a data discovery response from a TAP query or one of the simple
 DAL query protocols, to enable clients to find and use the \blinks\
-capability itself.
-
-Here we describe how to construct a resource that describes a service
-and add it to a VOTable document. The mechanism is general and can be
-used wherever a VOTable document is created.
-
+service directly.
```

Comment:
I think this text could be made more general: it says that the service descriptor mechanism can be used in any VOTable to enable clients to find a {links} service, but in fact it can be used in any VOTable to enable clients to find all kinds of services.